dts/arm/renesas/ra/ra8/r7fa8d1bhecbd.dtsi

```diff
@@ -25,5 +25,12 @@
 				renesas,programming-enable;
 			};
 		};
+
+		sdram: sdram@68000000 {
```

Comment:
Again here- should this be at the SOC level? Seems like the SDRAM on the EK-RA8D1 is an external memory chip specific to that board. Perhaps the reg address can be set here, but the size probably needs to be 0, and can be overridden at the board level.

Reply:
Done

Reply:
I think the best option here is probably one of these two approaches:

- Define the *entire* SDRAM node at the board level
- Define a node for the SDRAM controller on the SOC at the SOC level, and define the SDRAM part itself at the board level. This is similar to how we define other external devices, like flash chips on a FlexSPI bus.

Reply:
Done. I have updated as first one. Update for SDRAM controller as second one will be added later. Thanks for your idea.